src/pages/Home/index.js

```diff
@@ -1,9 +1,10 @@
 import React, { useEffect } from 'react';
-import ajax from '../../utils/ajax';
+import ajax from 'utils/ajax';
```

Comment:
You can use import like this, without so many dots.

Reply:
The base url is `/src`.

To change the base url, see /jsconfig.json for more detail.